src/Stripe.net/Entities/Cards/Card.cs

```diff
@@ -89,7 +101,7 @@ internal object InternalCustomer
         /// <summary>
         /// Whether this object is deleted or not.
         /// </summary>
-        [JsonProperty("deleted")]
+        [JsonProperty("deleted", NullValueHandling=NullValueHandling.Ignore)]
         public bool? Deleted { get; set; }
```

Comment:
Could we have `IStripeDeleted` interface or similar to ensure the setup of `Deleted` is always correctly done? Just worried we'll miss that custom setting in the future

Reply:
I'll add a wholesome test for this. To be clear, this is purely cosmetic: there is no functional difference between having `"deleted": null` or not having `"deleted"` at all in the JSON serialization.

Reply:
ohhhhh, wdyt of removing then?

Reply:
I think it's nice to keep the serialization fairly close to what the API would return, and the wholesome test should ensure we stay consistent as we add more deletable resources in the future.

Reply:
I disagree but I see your point so let's do it your way!